bundles/org.openhab.binding.mqtt.generic/README.md

```diff
@@ -167,12 +167,16 @@ The channel expects values on the corresponding MQTT topic to be in this format
 
 ### Channel Type "rollershutter"
 
-* __on__: An optional string (like "Open") that is recognized as UP state.
-* __off__: An optional string (like "Close") that is recognized as DOWN state.
-* __stop__: An optional string (like "Stop") that is recognized as STOP state.
+* __on__: An optional string (like "Open") that is recognized as `UP` state.
+* __off__: An optional string (like "Close") that is recognized as `DOWN` state.
+* __stop__: An optional string (like "Stop") that is recognized as `STOP` state.
+
+Internally `UP` is converted to 100%, `DOWN` to 0%.
```

Comment:
But this is exactly the opposite to what it is supposed to. Rollershutters are DOWN when at 100% and UP at 0% (see the definition in `UpDownType`).

Reply:
Code seems to be ok. Only the documentation is wrong.

Another thing (I did not touch it because it would be a breaking change): why is the config parameter for `UP` called `on` and for `DOWN` `off` instead of `up` and `down`?

Reply:
Good question, which probably only @davidgraeff can answer 🙄.

Reply:
Because the java class has an "on" and "off" field already, those simply got reused.
I guess it is possible to tell gson via annotations to map "up" and "down" to "on" and "off" fields to allow for both variants.